hack/test-script/asset.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: katalog.fybrik.io/v1alpha1
+kind: Asset
+metadata:
+  name: userdata
+spec:
+  secretRef: 
+    name: userdata
+  details:
+    dataFormat: csv
```

Comment:
does it work without specifying `csv`?

Reply:
no, I think that fybrik looks for a match between the dataformat in the catalog asset connection and the one in the module which includes:
```
        - source:
            protocol: mysql
            dataformat: csv